setup/1. intellij.md

```diff
@@ -0,0 +1,29 @@
+# IntelliJ and MC Development
+
+## IntelliJ
+
+IntelliJ is an IDE created by JetBrains. 
+IDE stands for Integrated Development Environment, and is software designed to make programming as easy as possible. 
+Features that IDEs often have are ways to compile all the code with the press of a button and to help complete code with valid suggestions. 
```

Comment:
Debugging with break/watchpoints and hot-swapping code are pretty important things too.